python/hopsworks_common/connection.py

```diff
@@ -429,7 +430,17 @@ def _provide_project(self, name=None):
         if self._variable_api.get_data_science_profile_enabled():
             # load_default_configuration has to be called before using hsml
             # but after a project is provided to client
-            self._model_serving_api.load_default_configuration()  # istio client, default resources,...
+            try:
+                # istio client, default resources,...
+                self._model_serving_api.load_default_configuration()
+            except RestAPIError as e:
+                if e.response.error_code == 403:
+                    print(
+                        "You lack permission to initialize model serving. Machine learning functionality will be not available."
+                    )
+                    print("The permission-related exception is:", e)
```

Comment:
Can you use a debug logger for that instead?

Reply:
I printed it for the same reason as is explained above. I will change it to use the debug logger.